cpu/cc26x0/Makefile.include

```diff
@@ -1,3 +1,5 @@
 export CPU_ARCH := cortex-m3
 
+USEMODULE += periph_common
```

Comment:
would be obsolete if we agree on #9906 😄

Reply:
Should it wait or can it be removed later?

Reply:
depends, what gets merged first 😄

Reply:
Good good.